transitions/accept_patient_reports.js

```diff
@@ -13,52 +13,53 @@ const _hasConfig = (doc) => {
     return Boolean(getConfig(doc.form));
 };
 
+// This is more complicated than it needs to be because JS / _ always use ===
+// for equality, complicating the use of unique tuples (i.e. [1,2] !== [1,2])
+const hasGroupTypeCombo = (groupTypeColl, [group, type]) =>
```

Comment:
I think `hasGroupAndType` might be a better name?

Reply:
Sure.